test/double_bucket_queue.cc

```diff
@@ -72,6 +83,16 @@ TEST(DoubleBucketQueue, TestClear) {
   TryClear(costs);
 }
 
+TEST(DoubleBucketQueue, RC4FloatPrecisionErrors) {
+  // Tests what happens when the internal floats in DoubleBucketQueue loses
+  // precision
+
+  std::vector<uint32_t> costs = {1320209856};
```

Comment:
Whats this value? Also, why do run sort on the single element array (does it trigger a bug)?

Reply:
Yes, the testcase is one generated from the property test RNG in https://github.com/valhalla/valhalla/compare/anders-double-bucket-prop-test

The sort is not important, it was simply the same pattern that the rest of the tests were following.